src/software/ai/passing/pass_generator_test.cpp

```diff
@@ -164,12 +142,11 @@ TEST_F(PassGeneratorTest, check_passer_robot_is_ignored_for_friendly_capability)
     UNUSED(converged_score);
 }
 
-TEST_F(PassGeneratorTest, check_pass_does_not_converge_to_self_pass)
+TEST_F(PassGeneratorTest, DISABLED_check_pass_does_not_converge_to_self_pass)
```

Comment:
why disabled?

Reply:
ah that slipped through, re-enabled 2 and delete 1 of them because its no-longer relevant